dev-server.js

```diff
@@ -23,15 +25,23 @@ app.use(require('webpack-dev-middleware')(compiler, {
 
 app.use(require('webpack-hot-middleware')(compiler, {log: () => {}}));
 
+// Setup proxy routes to use on Hosts that only expose one external port (i.e.
+// Heroku)
+var config = JSON.parse(fs.readFileSync('public/config.json'));
+app.use('/xenia', proxy(config.xeniaHost));
```

Comment:
nope, cay will still use config.xeniaHost, etc. The value of those config values will be set to the proxy routes if the config has proxy set to true. Otherwise, everything will work how it normally works.